editor/components/block-list/style.scss

```diff
@@ -400,6 +406,7 @@
 	&:before {
 		content: '';
 		position: absolute;
+		left: 0;
```

Comment:
In my testing, this change alone is sufficient to resolve the issue without the addition of `clear` styles. Can you confirm?

Reply:
I think you're right, i remember that inserter wasn't wrapping correctly without the `clear` property, but I no longer can reproduce it and it actually wraps the text when there is aligned element and 2 separate paragraph elements after it.

Thanks for looking into it. I have removed unnecessary styles.